app/controllers/thredded/application_controller.rb

```diff
@@ -88,10 +88,17 @@ def pundit_user
       thredded_current_user
     end
 
+    # Returns the `@messageboard` instance variable.
+    # If `@messageboard` is not set, it first sets it to the topic with the slug or ID given by
+    # `params[:messageboard_id]`.
+    #
+    # @return [Thredded::Messageboard]
+    # @raise [Thredded::Errors::MessageboardNotFound] if the topic with the given slug does not exist.
```

Comment:
We're a custom error so that main app can handle it in a custom way. Currently the default behaviour is to render `exception.message` for these custom errors (https://github.com/thredded/thredded/blob/ffebb25398fdf4de95b1cf14e70421c16f58d544/app/controllers/thredded/application_controller.rb#L17-L24). Maybe there is a better way to do this?